### PR TITLE
docs: correct getting started version

### PR DESCRIPTION
**Pull request by kassianah**
https://app.shortcut.com/mineiros/story/5060/correct-getting-started-example-version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,8 @@ Basic usage for creating an IAM Policy granting full access to AWS Simple Storag
 
 ```hcl
 module "role-s3-full-access" {
-  source  = "mineiros-io/iam-policy/aws"
-  version = "~> 0.5.0"
-
-  name = "S3FullAccess"
+  source = "mineiros-io/iam-policy/aws"
+  version = "~> 0.5.2"
```
Comment (mariux) [inline README.md]: name was removed and alignment is broken. pleas pre review PRs accordingly before marking them as ready for review.. ;)

File: README.md [hunk 2]
```diff
@@ -84,7 +82,7 @@ See [variables.tf] and [examples/] for details and use-cases.
 
   Default is `{}`.
 
-- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(any)`)*<a name="var-module_depends_on"></a>
+- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(object)`)*<a name="var-module_depends_on"></a>
```
Comment (mariux) [inline README.md]: that is the wrong type.. please update from template

Comment (mariux) [reply README.md]: why is any converted to object?

**Review by mariux — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (kassianah) [reply README.md]: My bad, sorry

File: README.md (hunk 2 above)
Comment (kassianah) [reply README.md]: Terradoc used to accept any as a type, but doesnt anymore

Comment (kassianah) [reply README.md]: In newer documentation the type is `list(dependency)`

**Pull request closed without merging**